docs/configuration.md

```diff
@@ -1062,11 +1062,14 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.ui.showConsoleProgress</code></td>
-  <td>true</td>
+  <td>false</td>
   <td>
     Show the progress bar in the console. The progress bar shows the progress of stages
     that run for longer than 500ms. If multiple stages run at the same time, multiple
     progress bars will be displayed on the same line.
+    <br/>
+    <em>Note:</em> In non-shell environment, the default value of spark.ui.showConsoleProgress is true or
```

Comment:
I think it's true in shells by default (not non-shell environment)

Reply:
Sorry, I make a mistake.